packages/devextreme-react/src/core/helpers.ts

```diff
@@ -12,7 +12,7 @@ export function generateID(): string {
 }
 
 export class DoubleKeyMap<TKey1, TKey2, TValue> {
-  private _map: Map<TKey1, Map<TKey2, TValue>> = new Map();
+  readonly _map: Map<TKey1, Map<TKey2, TValue>> = new Map();
```

Comment:
i believe, this should be left as private, because with readonly, u will still be able to access _map and modify it by calling methods

Reply:
I agree with you. I initially left it as private, but the linter throws an error.
![image](https://github.com/DevExpress/DevExtreme/assets/89575880/eff23343-c3e0-4104-ba97-8a563dc508ee)

Reply:
I think Ivan meant `private readonly`:

```js
private readonly _map: Map<TKey1, Map<TKey2, TValue>> = new Map();
```